app/frontend/src/features/connections/friends/AddFriendButton.stories.tsx

```diff
@@ -2,7 +2,7 @@ import { action } from "@storybook/addon-actions";
 import { Meta, Story } from "@storybook/react";
 import { Empty } from "google-protobuf/google/protobuf/empty_pb";
 
-import { mockedService } from "../../../stories/__mocks__/service";
+import { mockedService } from "../../../stories/serviceMocks";
```

Comment:
import from "stories/serviceMocks" ?

Reply:
Good catch 👍  The guilt of relying on auto import fix too much, it's not that smart 😅